### ISSUE TITLE
CIPP Notifications - Email and Webhook not sending

### ISSUE THREAD

**Issue by mmccollum2** [KelvinTegelaar/CIPP#1123]
### Description

We've setup the notifications with an email address and a slack webhook that we've tested (Sending an email  from 365 to the email address (generatedemail@ourslack.slack.com), and using powershell to use Invoke-WebRequest and test the webhook (https://hooks.slack.com/*/*/*****). Those both make it to the slack channel. We've also enabled all notifications below the email/webhook settings to get everything!  I've attached a photo with fake info in the boxes.

![cipp-notifications](https://user-images.githubusercontent.com/13972038/190024159-5cb9b373-7201-455c-afbf-a62e8bc21e5b.png)

Thank you in advance for any tips you have for maybe specific logs we can check? If you'd prefer this posted in the CIPP discord then let me know and i'll circle back and post what the issue was and then close it up.

### Environment data

```PowerShell
Hosted in Azure with the automated script
CIPP 
Frontend 2.12.0
Backend 2.1.1
Tokens are happy and valid/current
```

**Comment by mmccollum2** [comment]
I'm fairly certain that based on the [troubleshooting page](https://cipp.app/docs/general/troubleshooting/), I can confirm that I have checked the documentation thoroughly and believe this to be an actual bug. Maybe I have missed somewhere on the website that lists specific and internal logging that can be checked to troubleshoot this issue.

**Comment by mmccollum2** [comment]
I confirm that I have checked the documentation thoroughly and believe this to be an actual bug.

**Comment by KelvinTegelaar** [comment]
Seems to be a user issue; Everything is working in both dev and prod. Try using https://webhook.site/ to test the webhook.

You can click on your avatar and "logbook" to find the CIPP logs. I've tested with removing standards/adding standards etc.

**Comment by MikeYEG** [comment]
Hey @KelvinTegelaar 
Are there any other troubleshooting steps we can take on this? 
What API type should we see in the logbook when a notification is sent?

Things I've done to try and get a notification to send:
- I've confirmed that both the webhook and the email are valid
- I've added and removed standards to a tenant
- I've created and disabled a 365 user
- Restarted the function app, waited 30 minutes, and tried the above steps again.

Appreciate any help/feedback on this.
